docs/content/guides/dagster/automated_pipelines.mdx

```diff
@@ -0,0 +1,157 @@
+
+In the example below, the sensor will materialize the dbt assets when the parent Airbtye assets materialize.
+```python
+from dagster import build_asset_reconciliation_sensor
+
+defs = Definitions(
+    assets=[airbyte_asset1, airbyte_asset2, dbt_asset1, dbt_asset2],
+    sensors=[
+        build_asset_reconciliation_sensor(
+            asset_selection=AssetSelection.assets(dbt_asset1, dbt_asset2),
+            name="asset_reconciliation_sensor",
+        ),
+    ],
+)
+```
+---
+## Some of my data should be updated, but not everything
+
+Let’s say you have a database with all your credit card transactions. When a return occurs, the original purchase record doesn’t change last week’s data, but a new transaction showing the return is added to the current day’s information.
+
+In this case, you want your data pipeline to include all the latest data, but it doesn’t make sense to continuously process data from two years or two days ago when you know it hasn’t changed. What you want is to process yesterday’s data and not waste compute on the rest of the data. 
+
+Dagster [partitions](https://docs.dagster.io/concepts/partitions-schedules-sensors/partitions#defining-partitioned-assets) do just that. A partition is a slice of your data; in this case, each partition represents a day of transactions. You can partition or ‘split’ your data based on whatever makes the most sense, by one or [multiple dimensions](https://docs.dagster.io/concepts/partitions-schedules-sensors/partitions#multi-dimensionally-partitioned-assets).  
+
+Partitions can be used with both schedules and sensors. You can use schedules to kick off a partitioned job to [update slices of data](https://docs.dagster.io/concepts/partitions-schedules-sensors/partitions#creating-schedules-from-partitioned-jobs). You can also trigger runs once a specific partition has been updated using asset sensors based on [partitioned materializations](https://docs.dagster.io/concepts/partitions-schedules-sensors/asset-sensors#triggering-runs-upon-partitioned-materializations).
+--- 
+## Dagster Automation Cheat Sheet
+
+Use this table as a quick reference when building out your data pipelines!
+
+| Name | Good for automating data pipelines that need to be refreshed:  | Asset Support | Op/Graph Support |
+| --- | --- | --- | --- |
+| Basic and Cron scheduling | by kicking off updates at a predictable time interval | ✅ | ✅ |
+| Sensors | when a specific event occurs | ✅ | ✅ |
+| Asset sensors | after an asset or set of assets materialize  | ✅ |  |
+| Asset reconciliation sensors* | when assets are out of date with other assets | ✅ |  |
+| Freshness policy sensors* | when assets are beyond an acceptable lag time | ✅ |  |
+*Asset reconciliation sensors and freshness policy sensors can be used independently or be used for declarative scheduling.
```

Comment:
If you’re using the asset reconciliation sensor, you’re using declarative scheduling - there’s no way to use the asset reconciliation sensor that we would not consider “declarative scheduling”

Reply:
ok maybe ill just remove this comment?